src/Core/Jobs/BaseJobsHostedService.cs

```diff
@@ -35,7 +38,18 @@ public virtual async Task StartAsync(CancellationToken cancellationToken)
         {
             var factory = new StdSchedulerFactory(new NameValueCollection
             {
-                { "quartz.serializer.type", "binary" }
+                {"quartz.serializer.type", "binary"},
+                // Ensure each project has a unique instanceName
+                {"quartz.scheduler.instanceName", GetType().FullName},
+                {"quartz.scheduler.instanceId", "AUTO"},
+                {"quartz.jobStore.type", "Quartz.Impl.AdoJobStore.JobStoreTX"},
+                {"quartz.jobStore.driverDelegateType", "Quartz.Impl.AdoJobStore.SqlServerDelegate"},
+                {"quartz.jobStore.useProperties", "true"},
+                {"quartz.jobStore.dataSource", "default"},
+                {"quartz.jobStore.tablePrefix", "QRTZ_"},
+                {"quartz.jobStore.clustered", "true"},
+                {"quartz.dataSource.default.provider", "SqlServer"},
+                {"quartz.dataSource.default.connectionString", _globalSettings.SqlServer.ConnectionString},
```

Comment:
We'll want to create a new connection string specifically for Quartz or `GlobalSettings.SqlServer.JobSchedulerConnectionString` or something like that; simply because we'll be breaking this out to a separate database.